src/4_mockFetch.test.js

```diff
@@ -1,11 +1,63 @@
 import { fetchJoke } from './4_mockFetch';
+import fetchMock from 'fetch-mock-jest';
 
 // You'll practice how to mock Network requests (fetch)!
 // Simple: Mock fetch manually - https://www.youtube.com/watch?v=mHXhuPHiDj8
 // Real-world: Use `fetch-mock-jest` to mock fetch - https://www.youtube.com/watch?v=yhUep7E9O20
 // Bonus (Advanced): Use `msw` to intercept the requests. - https://www.youtube.com/watch?v=v77fjkKQTH0
-describe('4_mockFetch_test', () => {
-  it('...', () => {
-    // ...
+
+
+
+
+describe('fetchJoke', () => {
+  afterEach(() => {
+    fetchMock.reset();
+  });
+
+  test('should return a joke when called with a valid containsWord argument', async () => {
+    const containsWord = 'bug';
+    const mockResponse = {
+      type: 'single',
+      joke: 'Why was the JavaScript developer sad? Because he didn\'t know how to `null` his feelings.'
+    };
+
+    fetchMock.get(`https://v2.jokeapi.dev/joke/Programming?contains=${containsWord}`, mockResponse);
+
+    const result = await fetchJoke(containsWord);
+
+    expect(typeof result).toBe('string');
```

Comment:
question: Why did you assert the result typeof when you already assert below the `result` with toEqual?